sys/posix/pthread/pthread_attr.c

```diff
@@ -20,141 +20,145 @@
-    return -1;
+int pthread_attr_setschedparam(pthread_attr_t *attr,
+                               const struct sched_param *param) {
+  // TODO
+  (void)attr;
+  (void)param;
+  return -1;
 }
 
-int pthread_attr_setschedpolicy(pthread_attr_t *attr, int policy)
-{
-    // TODO
-    (void) attr;
-    (void) policy;
-    return -1;
+int pthread_attr_getschedpolicy(const pthread_attr_t *attr, int *policy) {
+  // TODO
+  (void)attr;
+  (void)policy;
+  return -1;
 }
 
-int pthread_attr_getinheritsched(const pthread_attr_t *attr, int *inherit)
-{
-    // TODO
-    (void) attr;
-    (void) inherit;
-    return -1;
+int pthread_attr_setschedpolicy(pthread_attr_t *attr, int policy) {
+  // TODO
+  (void)attr;
+  (void)policy;
+  return -1;
 }
 
-int pthread_attr_setinheritsched(pthread_attr_t *attr, int inherit)
-{
-    // TODO
-    (void) attr;
-    (void) inherit;
-    return -1;
+int pthread_attr_getinheritsched(const pthread_attr_t *attr, int *inherit) {
+  // TODO
+  (void)attr;
+  (void)inherit;
+  return -1;
 }
 
-int pthread_attr_getscope(const pthread_attr_t *attr, int *scope)
-{
-    // TODO
-    (void) attr;
-    (void) scope;
-    return -1;
+int pthread_attr_setinheritsched(pthread_attr_t *attr, int inherit) {
+  // TODO
+  (void)attr;
+  (void)inherit;
+  return -1;
 }
 
-int pthread_attr_setscope(pthread_attr_t *attr, int scope)
-{
-    // TODO
-    (void) attr;
-    (void) scope;
-    return -1;
+int pthread_attr_getscope(const pthread_attr_t *attr, int *scope) {
+  // TODO
+  (void)attr;
+  (void)scope;
+  return -1;
 }
 
-int pthread_attr_getstackaddr(const pthread_attr_t *attr, void **stackaddr)
-{
-    *stackaddr = attr->ss_sp;
-    return 0;
+int pthread_attr_setscope(pthread_attr_t *attr, int scope) {
+  // TODO
+  (void)attr;
+  (void)scope;
+  return -1;
 }
 
-int pthread_attr_setstackaddr(pthread_attr_t *attr, void *stackaddr)
-{
-    attr->ss_sp = (char *) stackaddr;
-    return 0;
+int pthread_attr_getstackaddr(const pthread_attr_t *attr, void **stackaddr) {
+  *stackaddr = attr->ss_sp;
+  return 0;
 }
 
-int pthread_attr_getstacksize(const pthread_attr_t *attr, size_t *stacksize)
-{
-    *stacksize = attr->ss_size;
-    if (*stacksize == 0) {
-        /* FIXME: the standard says that this function should return the default value if no explicit value was set. */
-    }
-    return 0;
+int pthread_attr_setstackaddr(pthread_attr_t *attr, void *stackaddr) {
+  attr->ss_sp = (char *)stackaddr;
+  return 0;
 }
 
-int pthread_attr_setstacksize(pthread_attr_t *attr, size_t stacksize)
-{
-    attr->ss_size = stacksize;
-    return 0;
+int pthread_attr_getstacksize(const pthread_attr_t *attr, size_t *stacksize) {
+  *stacksize = attr->ss_size;
+  if (*stacksize == 0) {
+    /* FIXME: the standard says that this function should return the default
+     * value if no explicit value was set. */
+  }
+  return 0;
+}
+
+int pthread_attr_setstacksize(pthread_attr_t *attr, size_t stacksize) {
+  attr->ss_size = stacksize;
```

Comment:
You can't change the size of a thread stack in RIOT

Reply:
Actually that line of code is untouched by me. Sorry it seems my formatter made review impossible I will fix this ASAP

Reply:
Okay the PR is now in a good state.
As you can see I have only combined the already existing implementations of `*stackaddr` and `*stacksize` into a combined implementation of `*stack`.